pkg/addons/addon_script_runner.go

```diff
@@ -0,0 +1,85 @@
+package addons
+
+import (
+	"fmt"
+
+	"encoding/json"
+	"github.com/xetys/hetzner-kube/pkg/clustermanager"
+	"io/ioutil"
+	"log"
+	"strings"
+	"time"
+)
+
+//ScriptRunnerAddon installs script runner
```

Comment:
Single space between `//` and the next part of comment (on all line)

```suggestion
// ScriptRunnerAddon installs script runner
```

Reply:
Although I added the suggested changes. I think this is personal preference, we should stick to what "go fmt" gives us to avoid wasting time on these trivial changes.

Reply:
@gadelkareem this is the suggested format for go stdlib like https://github.com/golang/go/blob/master/src/os/env.go

I'm not sure `go fmt` will process the comments, there are some issue related to that but the core team decide that comments should not be manipulated from fmt, see https://github.com/golang/go/issues/5128